docs/release_notes/ibexa_dxp_v4.6.md

```diff
@@ -0,0 +1,466 @@
+
+#### Focus mode
+
+With multiple changes to the Back Office UI intended to expose the most important information and actions, editors can now better focus on their work.
+For more information, see [Focus mode](https://doc.ibexa.co/projects/userguide/en/latest/getting_started/discover_ui/#focus-mode).
+
+![Focus mode](img/4.6_focus_mode.png "Focus mode")
+
+As part of this effort, some other changes were introduced that apply to both regular and Focus mode:
+
+- In Content item details view, tabs have been reordered by their relevance
+- **Authors** and **Sub-items** are now separate tabs in Content item details view
+- Former **Details** tab is now called **Technical details** and changed its position
+- Preview is available in many new places, such as the **View** tab in Content item details view, or as miniatures when you hover over the Content Tree
+
+![Sub-items tab](img/4.6_sub_items_tab.png "Sub-items tab")
+
+#### Site context selector
+
+With a drop-down list added to the top bar, which changes the site context, editors can choose that the Content Tree shows only those Content items that belong to the selected website.
+And if Content items belong to multiple websites but use different designs or languages depending on the SiteAccess settings, their previews also change.
+
+As part of this effort, the name of the "Sites" area of the main menu has changed to "Site management".
+
+![Site context selector](img/4.6_site_selector.png "Site context selector")
+
+#### Distraction free mode
+
+While editing Rich Text Fields, user can switch to distraction free mode.
+It expands the workspace to full screen and shows only editor toolbar.
+
+![Distraction free mode](img/4.6_distraction_free_mode.png)
+
+For more information, see [Distraction free mode](https://doc.ibexa.co/projects/userguide/en/master/content_management/create_edit_content_items/#distraction-free-mode).
+
+#### Simplified user actions
+
+Button text now precisely describes actions, so that users who create or edit content understand the purpose of each button.
+
+![Improved button text](img/4.6_publishing_options.png)
+
+#### Draft section added to Content
+
+For streamlining purpose, the **Draft** section is now situated under **Content**.
+Users can now easily find and manage their drafts and published content from one area.
+
+![Draft section added to Content](img/4.6_drafts.png)
+
+#### User profile and new options in user settings
+
+With personal touch in mind, editors can now upload their photos, and provide the following information in their user profiles:
+
+- Email
+- Department
+- Position
+- Location
+- Signature
+
+Also, editors and other users can customize their experience even better, with new configuration options that have been added to user settings.
+
+For more information, see [user profile and settings documentation]().
+
+#### Recent activity log
```

Comment:
I feel like this feature deserves a screenshot :)
We could also mention that new policies can be used to determine if admin users can see their own activities or everybody's. But maybe it's too specific for the release notes.